### ISSUE TITLE
Q-select not rendering value correctly from Chrome autofill

### ISSUE THREAD

**Issue by duongpc** [quasarframework/quasar#6342]
**Describe the bug**
The browser offers to autofill the form fields for you from the QInput and it puts a value into the input of the QSelect also, the value may not match any of the choices for the QSelect and the text remains in the QSelect's input field. 

**Codepen/jsFiddle/Codesandbox (required)**
https://codepen.io/chaos123456/pen/zYGGgEb

**To Reproduce**
Create an autofill profile in Chrome with address and province field filled, in the province field I provided "ON"
Click on Address bar, select address

**Expected behavior**
State field should show "Ontario"

**Screenshots**
![image](https://user-images.githubusercontent.com/24272450/74262777-831cdb80-4ccb-11ea-8cc0-20e54eb1aa02.png)

**Comment by hawkeye64** [comment]
I got the same:
![image](https://user-images.githubusercontent.com/10262924/74263044-477a1580-4cbb-11ea-889c-59018ed5a403.png)

**Comment by duongpc** [comment]
my current workaround is to use `@keyup = autofillcheck` event
```
autofillcheck: function (e) {
   if (e.target && e.target.value) this.state = e.target.value
}
```
the issue with this is if user's autofill value is "Ontario" instead of "ON" then it won't work. Also will have to add method/event whenever I want to support autofill dropdown.

**Comment by rstoenescu** [comment]
Enhancement will be available in "quasar" v1.9.0.

**Comment by hawkeye64** [comment]
Works great, except for 1 issue @rstoenescu 
![image](https://user-images.githubusercontent.com/10262924/74374848-787e4700-4d9c-11ea-8784-5c8e168a5a56.png)

**Comment by pdanpdan** [comment]
And do you have Alberta in the list? Can you make a codepen?

**Comment by duongpc** [comment]
@pdanpdan autofill doesnt work if I take out user-input

see new codepen:
https://codepen.io/chaos123456/pen/JjdYmdd

**Comment by rstoenescu** [comment]
Improvements on this will be available in "quasar" v1.9.3.
